cocos/2d/renderer/static-vb-accessor.ts

```diff
@@ -178,7 +178,7 @@ export class StaticVBAccessor extends BufferAccessor {
             if (entry) break;
         }
         // Allocation fail
-        if (!entry) {
+        if (!entry && vertexCount <= this._vCount) {
```

Comment:
I think should check it at the beginning of this function, for example
public allocateChunk (vertexCount: number, indexCount: number): StaticVBChunk | null {
    if (vertexCount > this._vCount) {
        return;
    }
    ...
}

Reply:
I feel it's appropriate to put it here as well. Because most of the cases will not exceed the size, and only a very small part of the cases will have errors. Putting it at the top doesn't improve performance.

Reply:
Add at top of the function will not affect performance for other situations but can improve performance in special cases, why not do it?

Reply:
It's a error that needs to be fixed. It currently needs to be fixed at the user, so performance is not really a concern in this case.

Reply:
ok